_opcodes/op271-bgee.html

```diff
@@ -0,0 +1,16 @@
+---
+n: 271
+name: "Graphics: Avatar Removal"
+param1: "Irrelevant"
+param2: "Irrelevant"
+bg1: 0
+bg2: 0
+bgee: 1
+iwd1: 0
+iwd2: 0
+pst: 0
+---
+Hides the creature's animation and selection circle, including illusionary images (Blur / Mirror Image).<br/>
+Disables the creature's AI scripts.<br/>
+The creature cannot take any action not listed in <a href="../files/ids/bgee/instant.htm">INSTANT.IDS</a> (usually through <a href="../scripting/actions/bgeeactions.htm#1">ActionOverride()</a>.<br/>
+The creature is still subject to area effect and may still be targeted directly by mouseover.
```

Comment:
Are you sure this one is different in the ees? It doesn't read like it, except perhaps the last line.

Reply:
> Are you sure this one is different in the EEs?

No, I'm not.
I mean, I do not own the original games, I can only verify EE behavior...